### ISSUE TITLE
reproduction of results

### ISSUE THREAD

**Issue by MichealRay** [danfenghong/IEEE_TGRS_SpectralFormer#4]
Hi danfeng,
    Very good job. Congraqulations! And thank you for sharing codes.
    I have a problem when I want to reproduce results of Houston2013. 
    The training detail log is shown as below: 
Running Time: 1737.75
**************************************************
Final result:
OA: 0.7679 | AA: 0.7864 | Kappa: 0.7486
[0.83095916 0.82894737 0.80594059 0.94886364 0.99621212 0.93706294
 0.80970149 0.51377018 0.74032106 0.44980695 0.66223909 0.79634966
 0.73333333 0.90688259 0.83509514]
**************************************************
Parameter:
dataset: Houston
mode: CAF
gpu_id: 0
seed: 0
batch_size: 64
test_freq: 5
patches: 7
band_patches: 3
epoches: 1000
learning_rate: 0.0005
gamma: 0.9
weight_decay: 0

And the result in your paper is shown as below:
![Uploading houston2013_result.png…]()

As you can see there is a big gap between OA: 0.7679 | AA: 0.7864 | Kappa: 0.7486 and OA: 0.8801 | AA: 0.8891 | Kappa: 0.8699

So what I miss?  Could you please help me?

**Comment by YunyanYao** [comment]
@MichealRay  Thanks for your interest. You can help yourself by spending more time reading the implementation details in the paper.

**Comment by YunyanYao** [comment]
@MichealRay Thanks for your interest. You can help yourself by spending more time reading the implementation details in the paper. More supplementary files will be uploaded soon. Please kindly keep an eye on our repository.

**Comment by YunyanYao** [comment]
Hi, @MichealRay . Thanks for your interest. You can help yourself by spending more time reading the implementation details in the paper. More supplementary files will be uploaded soon for the inexperienced to reproduce all the results. Please kindly keep an eye on our repository.
